### PR TITLE
Helm chart refactor

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (asafMasa) [inline .vscode/settings.json]: fix conflicts

File: helm/templates/_resources.tpl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Create deployment name as used by the service name label.
+*/}}
+{{- define "cronjob.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "cron-job" | indent 1 }}
+{{- end }}
+
+{{/*
+Create route name as used by the service name label.
+*/}}
+{{- define "route.fullname" -}}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: no need for route

File: helm/templates/_resources.tpl [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: no need for ingress

File: helm/templates/_resources.tpl [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: end of file

File: helm/templates/_resources.tpl [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{{/*
+Create service name as used by the service name label.
+*/}}
+{{- define "service.fullname" -}}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: do we need a service?

File: helm/templates/cron-job.yaml [hunk 6]
```diff
@@ -57,51 +67,51 @@ spec:
...
                 - name: NODE_EXTRA_CA_CERTS
-                  value: {{ printf "%s/%s" .Values.rasterCommon.ca.path .Values.rasterCommon.ca.key | quote }}
+                  value: {{ printf "%s/%s" .Values.global.ca.path .Values.global.ca.key | quote }}
                 {{- end }}
               envFrom:
                 - configMapRef:
-                    name: {{ $releaseName }}-configmap
+                    name: {{ $configmapName }}
               volumeMounts:
                {{- if or .Values.env.cleanupTypes.failedIngestionTasks .Values.env.cleanupTypes.successfulIngestion }}
```
Comment (asafMasa) [inline helm/templates/cron-job.yaml]: please add or "successfulSwapUpdate"

**Review by asafMasa — CHANGES_REQUESTED** [review]
please merge master to this branch

File: .vscode/settings.json (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply .vscode/settings.json]: merged and fixed

File: helm/templates/_resources.tpl (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: removed

File: helm/templates/_resources.tpl (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: removed

File: helm/templates/_resources.tpl (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: fixed

File: helm/templates/_resources.tpl (hunk 5 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: nope, removed

File: helm/templates/cron-job.yaml (hunk 6 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/cron-job.yaml]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**